Adding health and readiness probes to Kubernetes documentation (#410)
spiceai/docs: spiceaidocs/docs/deployment/kubernetes/index.md

Once the monitoring is enabled, import the [Spice Grafana dashboard](../../clients/grafana/index.md) to visualize the Spice.ai metrics.

> ### Health and Readyness

Reviewer: ```suggestion
### Health and Readiness
```

Also worth providing a link to K8s docs: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes

> Spice supports three endpoints to help you monitor the state of your Spice: `/health` and `/ready`:

Reviewer: Avoid the use of "we" "I" "you".

```suggestion
Spice provides two HTTP endpoints to help monitor the state of the Spice runtime: `/health` and `/ready`:
```

#### Health probe

The `/health` endpoint indicates whether or not the Spice process is up and running and ready to receive requests. You can add a probe to query this endpoint like so:

```yaml
livenessProbe:
  httpGet:
    path: /health
    port: 8090
```

In Kubernetes, this pod will not be marked as *Healthy* until the `/health` endpoint returns `200`.

> #### Readyness probe

Reviewer: ```suggestion
#### Readiness probe
```

Once the monitoring is enabled, import the [Spice Grafana dashboard](../../clients/grafana/index.md) to visualize the Spice.ai metrics.

### Health and Readiness

Spice provides two HTTP endpoints to help monitor the state of the Spice runtime: `/health` and `/ready`:

#### Health probe

The `/health` endpoint indicates whether or not the Spice process is up and running and ready to receive requests. You can add a probe to query this endpoint like so:

```yaml
livenessProbe:
  httpGet:
    path: /health
    port: 8090
```

In Kubernetes, this pod will not be marked as *Healthy* until the `/health` endpoint returns `200`.

#### Readiness probe

> The `/ready` endpoint indicates **whether or not the Spice datasets are ready**. This means that while `/health` might indicate that Spice is up and running, until `/ready` reports a status of `200`, your queries may return 0 results

Reviewer 2: It should be `/v1/ready`
